### PR TITLE
Linted

### PR DESCRIPTION
**Pull request by evaherrada**

### CODE REVIEW

**Comment by evaherrada** [comment]
had to disable unspecified-encoding in a few spots. Looked into it a bit, and to be honest I feel like we should get rid of the check org-wide. I've come across it in a few places and every time I came across it I ended up disabling it.

File: examples/featherwing_keyboard_featherwing.py [hunk 1]
```diff
@@ -20,13 +20,16 @@
 kbd_featherwing.neopixel[0] = 0x002244
 
 try:
-    f = open("/sd/tft_featherwing.txt", "w")
-    f.write("Blinka\nBlackberry Q10 Keyboard")
-    f.close()
+    with open(  # pylint: disable=unspecified-encoding
+        "/sd/tft_featherwing.txt", "w"
```
Comment (kattni) [inline examples/featherwing_keyboard_featherwing.py]: I'm a bit confused here - it was my understanding that the `"w"` is the encoding. Same for the `"r"` below. I don't understand why Pylint is throwing this error here.

Comment (kattni) [reply examples/featherwing_keyboard_featherwing.py]: Wait no, that's mode. You may be right about disabling org-wide. I'll think on it.

**Comment by kattni** [comment]
I'm also wondering why it's not failing on other libraries that have `open()` without encoding specified.

**Comment by evaherrada** [comment]
@kattni I think they are now. Pretty sure it's just because we changed the pylint version. It's a pretty recent one

**Comment by kattni** [comment]
I was referring to a few that I merged earlier that were not disabled, but the PR passed. Either way, we're disabling globally at this point. So it's fine.

**Review by kattni — CHANGES_REQUESTED** [review]
Can you please remove the `unspecified-encoding` disables?

**Comment by evaherrada** [comment]
Sure

**Comment by kattni** [comment]
@dherrada I'm going to give the contributor on #74  an opportunity to integrate these changes into their PR before merging this one over theirs. Please still remove the disable I mentioned in my last comment, so they have the appropriate set of changes to work with. If they're not interested in integrating into their PR, I'll merge this at that point.

**Comment by evaherrada** [comment]
Ah ok

File: examples/featherwing_tft35_simpletest.py [hunk 2]
```diff
@@ -13,13 +13,15 @@
 tft_featherwing = tft_featherwing_35.TFTFeatherWing35()
 
 try:
-    f = open("/sd/tft_featherwing.txt", "w")
-    f.write("Blinka\nBlackberry Q10 Keyboard")
-    f.close()
+    with open(  # pylint: disable=unspecified-encoding
```
Comment (kattni) [inline examples/featherwing_tft35_simpletest.py:16]: @dherrada You missed two :)

**Comment by kattni** [comment]
Closing in favor of #74

**Pull request closed without merging**